### PR TITLE
handle type any as complex

### PR DESCRIPTION
**Pull request by changlong-liu**
for fix az issue #210

### CODE REVIEW

File: src/plugins/azgenerator/CodeModelAzImpl.ts [hunk 1]
```diff
@@ -686,6 +686,13 @@ export class CodeModelCliImpl implements CodeModelAz {
         return this.MethodParameter.schema.type;
     }
 
+    private isComplexSchema(type: string): boolean {
+        if (type == SchemaType.Array || type == SchemaType.Object || type == SchemaType.Dictionary || type == SchemaType.Any)
```
Comment (qiaozha) [inline src/plugins/azgenerator/CodeModelAzImpl.ts]: Can you merge master into your branch? I think Rodge has already done some changes about type any before.
I think you can keep this, but add one more condition into it. when a parameter is manually set as json, this should also be treated as ComplexSchema. right?

Comment (changlong-liu) [reply src/plugins/azgenerator/CodeModelAzImpl.ts]: Hi, just merged and add manual json setting check.

File: src/plugins/azgenerator/CodeModelAzImpl.ts [hunk 2]
```diff
@@ -790,10 +804,12 @@ export class CodeModelCliImpl implements CodeModelAz {
         if (this.MethodParameter_IsFlattened) {
             return false;
         }
+        
         if (this.MethodParameter.language['cli'].json == true) {
             return true;
         }
```
Comment (qiaozha) [inline src/plugins/azgenerator/CodeModelAzImpl.ts]: This can also be removed now. right?

Comment (changlong-liu) [reply src/plugins/azgenerator/CodeModelAzImpl.ts]: yes, just removed

**Pull request merged**